source/_integrations/zha.markdown

```diff
@@ -18,6 +18,7 @@ ha_codeowners:
   - '@dmulcahey'
   - '@adminiuga'
 ha_domain: zha
+toc: true
```

Comment:
Why here and not elsewhere?

Reply:
I just set it as an example for the review app. This will need to be added to all integrations, not sure if there's a way to set it by default once and for all :thinking:

Reply:
Added to all.